apps/css-workshop/src/pages/radio.astro

```diff
@@ -388,45 +224,27 @@ import Icon_ from '../components/Icon.astro';
         <span class='iui-radio-label'>Option 1</span>
         <input class='iui-radio' type='radio' name='radio-demo-12' value='option-1' checked />
       </label>
-
       <label class='iui-radio-wrapper'>
         <span class='iui-radio-label'>Option 2</span>
         <input class='iui-radio' type='radio' name='radio-demo-12' value='option-2' />
       </label>
```

Comment:
Is there no way to handle radio on the right hand side with the Astro component?

Reply:
I have added a flag prop called `isFlipped` (I feel like it should have a better name) so the `input` and `span` label can be swapped in the radio on right layout. Not sure if this is the right way to do it.

Reply:
Since these Astro components are for our use only I am ok with that prop name, but I'll leave that decision up to @mayank99.

Reply:
In `ToggleSwitch`, we call it `labelPosition` so we could reuse that name.

https://github.com/iTwin/iTwinUI/blob/c977e9f39d3c05ce8278d0d3b72df3a7c5ee662a/packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx#L23

As for the code, I think the `<input>` can stay where it is. Only the `<span>` needs to be conditionally rendered before or after the `<input>`.